shared/src/common/types/search/search-payload.type.ts

```diff
@@ -0,0 +1,7 @@
+import { SearchPayloadKey } from '~/common/enums/enums';
+
+type SearchPayload = {
```

Comment:
Let's rename it to the `PodcastSearchPayload`
(don't forget about the file name, enum etc)

Reply:
Done